util/Roles.py

```diff
@@ -0,0 +1,12 @@
+from util.JsonHandler import *
+
+
+roles = load_json("roles.json")
+
+def isModerator(ctx):
+    if(ctx.guild == None): return False
+    for roleID in roles['Moderator']:
+        # check if any of the roles the user has is Moderator
+        if(any(map(lambda x : x.id == int(roleID), ctx.author.roles)) == True):
```

Comment:
There is probably a better way to check this but as I'm a novice in Python IDK if something similar to javascript's `some`or C#'s `Any` exists